stanford-earth/matson
# Earth 1450  search updates

# READY to test, Not merge, contains console logs still

# Summary
- Adjusts js and scss

# Needed By (Date)
- 9/11

# Steps to Test

1. compile js and scss
2. commit changes
3. we need to test on a deploy to see that search hovers, expands, and closes correctly. We don't want to see overlapping text labels when search is open, and the search icon should always show correctly and not disappear.
Test the homepage and the search pages.

# Affected versions or modules
- Does this PR impact any particular module, version, or pull request?

# Associated Issues and/or People
- JIRA ticket SE3:-
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
